lib/hanami/helpers/html_helper/empty_html_node.rb

```diff
@@ -129,6 +129,21 @@ def boolean_attribute(attribute_name, _value)
         def attribute(attribute_name, value)
           %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{value}")
         end
+
+        # Changes any attributes values that are arrays into a string joined by
+        # a space, as HTML expects it.
+        #
+        # @param attributes [Hash] the attributes with values to be HTMLified
+        #
+        # @return [Hash] the attributes transformed into HTML friendly values, i.e space separated strings
+        #
+        # @since x.x.x
+        def htmlified_array_attributes(attributes)
+          attributes&.inject({}) do |attrs, (key, value)|
```

Comment:
There are a number of ways to do this, including `.map.to_h`, but I chose what I think is the most obvious one.

Ruby 2.4 has `Hash#transform_values`, which we can leverage once we stop supporting Ruby 2.3. (Should I add that as a `TODO:` comment?)